Automattic/jetpack
# Lazy Images: Prefer Jetpack lazy loading over loading attribute tag

Fixes #23553

#### Changes proposed in this Pull Request:

While debugging #23553, I observed that in `master`, when I loaded a page with many images and then scrolled, that the images outside the viewport would load but the images that were initially in the viewport would not load.

This led me to think that there was some conflict between Safari's implementation and our JavaScript implementation. As I thought more about it, it doesn't make much sense to me to use a JavaScript implementation of lazy images and the core browser implementation. So, this PR chooses to solve the conflict by removing the `loading` tag when initializing an image for lazy loading via Jetpack.

#### Jetpack product discussion



N/A

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






* Checkout PR
* Create a post with multiple images in a gallery, enough to create multiple rows ideally
* Try to load the post starting with different viewport sizes and ensure that images always load
